fedimint-bin-tests/src/main.rs

```diff
@@ -502,13 +502,38 @@ async fn latency_tests(dev_fed: DevFed) -> Result<()> {
             .out_json()
             .await?;
         let invoice = json_encode(&invoice["invoice"]);
-        // what is awk print $30 ?
+        let payment = lnd
+            .client_lock()
+            .await?
+            .send_payment_sync(tonic_lnd::lnrpc::SendRequest {
+                payment_request: invoice,
+                // no equivalent of --force?
```

Comment:
On the command line, if you don't pass `--force`, it asks you to confirm that payment by typing `yes / no` which isn't suitable for scripts. This doesn't happen at all via RPC, so you can remove this comment.